test/test_ad5754r.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+hardware = "ad5754r"
+classname = "adi.ad5754r"
+
+
+#########################################
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [0])
+def test_ad5754r_tx_data(test_dma_tx, iio_uri, classname, channel):
+    # TODO add function call once driver is updated with waveform generation
+    # capabilities
```

Comment:
Need to wait for the driver to make sure this actually works. Can you enhance this test code to include the other properties as well? We are trying to increase general test coverage

Reply:
I've removed this for now, and have added other attribute tests.

Reply:
Is there a way to test string properties of the channel class? I can see existing test methods for integer properties of channel classes, but not for non-integer ones.

Reply:
You would need to enhance the `attribute_single_value_str` function to support sub-channels

Reply:
I see, so should I leave the test script the way it is now, with tests for the global attributes? I will be revisiting this later anyway when I add support for waveform generation, so if there is support added for channel string property tests by, I could use those with my driver?